_partials/self-hosted/_setup-steps.mdx

```diff
@@ -0,0 +1,243 @@
+
+    ```text hideClipboard
+    Setting up SSL Certs
+    /opt/spectro/functions.sh: line 118: /etc/nginx/.htpasswd: No such file or directory
+    chmod: cannot access '/etc/nginx/.htpasswd': No such file or directory
+    mkdir: cannot create directory ‘/etc/nginx/ssl’: No such file or directory
+    cp: target '/etc/nginx/ssl' is not a directory
+    Setting up Harbor
+    setenforce is /usr/sbin/setenforce
+    Setup Completed
+
+    Details:
+    -------
+    Spectro Cloud Repository
+    Location: https://palette.example.com:8443
+    UserName: spectro
+    Password: **************
+    CA certificate filepath: /opt/spectro/ssl/server.crt
+
+    Pack OCI Registry
+    Endpoint: https://palette.example.com
+    Base Content Path: spectro-packs
+    CA certificate Filepath: /opt/spectro/ssl/server.crt
+    Username: admin
+    Password: **************
+
+    Image OCI Registry
+    Endpoint: https://palette.example.comv
+    Base Content Path: spectro-images
+    CA certificate Filepath: /opt/spectro/ssl/server.crt
+    Username: admin
+    Password: **************
+    ```
+
+16. Update the SSL certificate file and key in the **httpd** service.
+
+    :::warning
+
+    You can skip this step if you do not plan to use the local Spectro Cloud Artifact Repository (SCAR) during the
+    [{props.edition} installation](#next-steps) process.
+
+    :::
+
+    Open the `/etc/httpd/conf.d/ssl.conf` file and add the path to the certificate and key generated in step **15** of
+    this guide:
+
+    - Replace the line `SSLCertificateFile /etc/pki/tls/certs/localhost.crt` with
+      `SSLCertificateFile  /opt/spectro/ssl/server.crt`.
+    - Replace the line `SSLCertificateKeyFile /etc/pki/tls/private/localhost.key` with
+      `SSLCertificateKeyFile /opt/spectro/ssl/server.key`.
+
+    Save and exit the file. Next, restart the HTTP server to apply the changes.
+
+    ```shell
+    systemctl restart httpd.service
+    ```
+
+17. Start the {props.edition} installation binary, which uploads release-specific {props.edition} images and packs to the Harbor registry.
+
+    ```shell
+    chmod +x airgap-<version>.bin && ./airgap-<version>.bin
+    ```
+    
+    Consider the following example for reference.
+
+    ```shell
+    chmod +x airgap-4-4-14.bin && ./airgap-4-4-14.bin
+    ```
+
+    This step may take some time to complete. A `Setup Completed` message confirms it is finished.
+
+    ```text hideClipboard {8}
+    Verifying archive integrity...  100%   MD5 checksums are OK. All good.
+    Uncompressing Airgap Setup - Version 4.4.14  100%
+    Setting up CLI
+    Setting up Manifests
+    Setting up Packs
+    ...
+
+    Setup Completed
+    ```
+
+18. Grant the Apache user and group the necessary permissions to serve the files in the `/var/www/html` directory.
+
+    ```shell
+    chown -R apache.apache /var/www/html
+    ```
+
+    Restart the HTTP server to apply the changes.
+
+    ```shell
+    systemctl restart httpd.service
+    ```
```

Comment:
In the OVA guide we mention supplemental packs and next steps, along with a warning about restarting the instance. Can we do something similar here as well?

![CleanShot 2024-10-16 at 15 05 21](https://github.com/user-attachments/assets/3cf5e641-fb54-4c2f-b0f1-2b8c9d00003e)

Reply:
Oh, yes! Great catch. I added the steps :)